examples/digitalocean/do/outputs.tf

```diff
@@ -10,3 +10,6 @@ output "addresses" {
   value = digitalocean_droplet.rke-node[*].ipv4_address
 }
 
+output "internal_addresses" {
+  value = digitalocean_droplet.rke-node[*].ipv4_address
```

Comment:
`internal_address` points `ipv4_address` (not `ipv4_address_private`).
Is this what you intended?

Reply:
oops, it has to be ipv4_address_private